### PR TITLE
PR: Simplify new terminal actions

### PR DESCRIPTION
**Pull request by ccordoba12**
* I found very inconvenient to have a submenu to access new terminal actions, so I removed it.
* I renamed the text of each action to be consistent with the wording used in Spyder.
* This PR also allows users to rename terminal tabs, a new feature of Spyder 3.2 which I find very convenient :-)

### CODE REVIEW

File: spyder_terminal/terminalplugin.py [hunk 1]
```diff
@@ -190,22 +191,20 @@ def get_plugin_icon(self):
...
-        new_terminal_menu.setIcon(ima.icon('project_expanded'))
         new_terminal_cwd = create_action(self,
-                                         _("Current workspace path"),
-                                         icon=ima.icon('cmdprompt'),
+                                         _("New terminal in current "
+                                           "working directory"),
                                          tip=_("Sets the pwd at "
-                                               "the current workspace "
-                                               "folder"),
+                                               "the current workind "
```
Comment (andfoy) [inline spyder_terminal/terminalplugin.py]: We have a minor typo here

Comment (andfoy) [inline spyder_terminal/terminalplugin.py]: We have a minor typo here

**Comment by goanpeca** [comment]
@ccordoba12 could you paste a screenshot :-p?

**Comment by ccordoba12** [comment]
**Before**:

seleccion_002

**After**:

seleccion_003

**Comment by ccordoba12** [comment]
Glad you like it!

**Pull request merged**